app/dashboard/views.py

```diff
@@ -837,7 +837,28 @@ def profile(request, handle):
         'stats': profile.stats,
         'bounties': profile.bounties,
         'tips': Tip.objects.filter(username=handle, network='mainnet'),
+        # TODO: Fill following with real data
```

Comment:
👍 I'll tackle this for you once I'm back from pycon this evening.

Reply:
Sorry for the delay here.  I'm adding this to my list for tomorrow.
Thanks!

Reply:
is this in? i still see the TODO on the branch